content/22.vaccines.md

```diff
@@ -405,6 +405,116 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
+As of March 6th, 2021, approximately 319 million vaccine doses have been administered in at least 118 countries worldwide using 10 different vaccines [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html; @url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/]. 
+The global vaccination rate is currently ~8.1 million doses per day, which at the current rate would take almost 4 years to vaccinate 75% of the world's population according to media estimates of a two-dose regimen [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/].
+Vaccine production and distribution varies from region to region and seems to depend on the availability of the vaccines and potentially a country's resources and wealth [@url:https://www.nytimes.com/2020/12/25/business/coronavirus-vaccines-global-economy.html].
+
+In North America, the majority of vaccines distributed until March 2021 have been produced by Pfizer-BioNTech and Moderna. 
+In Canada, the vaccine approval process is conducted by Health Canada who use a fast-tracked process whereby vaccine producers can submit data as it becomes available to allow for rapid review.
+An approval is granted following reviews of the available phase III clinical data. 
+This is followed by a period of pharmacovigilance in the population using their post-market surveillance system, which will monitor the long-term safety and efficacy of any vaccines [@url:https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/vaccines/development-approval-infographic.html; @url:https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/covid-19-vaccine-treatment/safety-after-authorization.html].
+Health Canada have authorized the use of the Pfizer (December 9th, 2020), Moderna (December 23rd, 2020), Oxford-AstraZeneca (February 26th, 2021), and the Janssen (March 5th, 2021) vaccines and the Novavax Inc vaccine is also under consideration [@url:https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/authorization/applications.html#wb-auto-4].  
+While Canada initially projected that by the end of September 2021 a vaccine would be available for all Canadian adults, they now predict that it may be possible earlier as more vaccines have been approved and become available [@url:https://www.cbc.ca/news/politics/trudeau-possible-vaccination-campaign-ends-sooner-1.5934994]. 
+
+In the U.S., vaccines are required to have demonstrated safety and efficacy in phase III trials before manufacturers apply for an emergency use authorization (EUA) from the FDA. 
+If an EUA is granted, an additional evaluation of the safety and efficacy of the vaccines is conducted by the CDC’s Advisory Committee on Immunization Practices (ACIP) who also provide guidance on vaccine prioritization. 
```

Comment:
The process is not very well documented actually because they never had to do it for such speed before. I think the CDC now has an information page that they keep adding to that gives an account of the process. I will find it and cite it in the next update of this section.